Makefile.am

```diff
@@ -630,15 +619,29 @@ if ENABLE_MPI
     # users to link to both libgeopm and libgeopmpolicy when calling
     # into application facing APIs.
     libgeopm_la_SOURCES = $(libgeopmpolicy_la_SOURCES) \
+                          src/DefaultProfile.cpp \
                           src/MPIComm.cpp \
                           src/MPIComm.hpp \
                           src/MPICommSplit.cpp \
+                          src/Profile.cpp \
+                          src/Profile.hpp \
                           src/geopm_ctl.h \
                           src/geopm_mpi_comm_split.h \
                           src/geopm_pmpi.c \
                           src/geopm_pmpi_helper.cpp \
                           src/geopm_pmpi.h \
                           # end
+
+if ENABLE_OMPT
+    libgeopm_la_SOURCES += src/OMPT.cpp \
```

Comment:
would be nice, in theory to have unit test for OMPT.cpp that didn't require openmp.  There is no explicit dependence on OpenMP in the OMPT class.  It is all in the ompt_callback.cpp.